admin/add_copyright.py

```diff
@@ -58,12 +57,12 @@ def getLicenseSection(filename, comment_char="#"):
     yield comment_char + HEADER_END
 
 
-def addCopyrightToPythonFile(filename, lines):
+def addCopyrightHeaderToPythonFile(filename, lines):
     # Empty file should be kept empty
     if len(lines) == 0:
         return lines
 
-    newLines = []
+    newLines = list([])
```

Comment:
The expressions `[]`, `list()`, and `list([])` all produce the equivalent result: an empty list. Thus, I'd usually use a list literal `[]`, which is also the shortest.

Reply:
Ok. I thought for typing it's better to write the long form. I'll change it.